lib/utils/logger.js

```diff
@@ -119,7 +122,7 @@ const log = method => (message, deviceId) => {
 			: stColors.suit(devicePrefix(deviceId)) + prefix + message
```

Comment:
@jan-tomasek-iflix could you please move `currentTimestamp` to lines 121 and 122, see that it's prepended before "Device XXXXX>" string